packages/xdl/src/Project.ts

```diff
@@ -951,9 +951,15 @@ async function _uploadArtifactsAsync({
   formData.append('iosBundle', iosBundle, 'iosBundle');
   formData.append('androidBundle', androidBundle, 'androidBundle');
   formData.append('options', JSON.stringify(options));
-  let response = await Api.callMethodAsync('publish', null, 'put', null, {
-    formData,
-  });
+
+  let response: any;
+  if (process.env.EXPO_LEGACY_API) {
```

Comment:
can we invert this so we have a flag on `EXPO_NEXT_API` or something and then we can land this and keep using the current/legacy api for now?